blueprints/in-repo-engine/routable-files/lib/__name__/addon/templates/application.hbs

```diff
@@ -1 +1 @@
-{{outlet}}
+{{outlet}}
```

Comment:
What's the reasoning for removing the new line? I just ask since most projects I work on have a style rule to have the new line.